### PR TITLE
Lable fix for LDAP

### PR DESCRIPTION
**Pull request by CamoMacdonald**
## Description

Corrected the labelling for openLDAP specifically to make it easier to configure.
## Motivation and Context

Took ages to figure out correct value for openLDAP as help descriptions said old value
## How To Test This

view help labels in Admin > Password management > LDAP
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

**Comment by horus68** [comment]
@CamoMacdonald Maybe you should consider to make the same change in /modules/Configurator/language/en_us.lang.php so values will be exactly the same (to make Translators life easier!)
See:
```
LBL_LDAP_ADMIN_USER_DESC
LBL_LDAP_AUTHENTICATION_DESC
LBL_LDAP_USER_FILTER_DESC
LBL_LDAP_BIND_ATTRIBUTE_DESC
LBL_LDAP_LOGIN_ATTRIBUTE_DESC
LBL_LDAP_AUTO_CREATE_USERS_DESC
```

https://github.com/salesagility/SuiteCRM/blob/e7fe1d56b62f650824cbbfca1fcd8da8dda46205/modules/Configurator/language/en_us.lang.php#L176

**Comment by daniel-samson** [comment]
@CamoMacdonald Can you respond to @horus68 ?

**Comment by likhobory** [comment]
'SHOW_DOWNLOADS_TAB'
'SHOW_DOWNLOADS_TAB_HELP'

Please provide some information about those labels, Downloads Tab has been used in Sugar interface: http://support.sugarcrm.com/Documentation/Sugar_Versions/6.5/Pro/Application_Guide/Getting_Started/#Downloads_Tab

But I can't find this feature in Suite interface.

File: modules/Configurator/language/en_us.lang.php [hunk 1]
```diff
@@ -130,45 +130,46 @@
...
-    'LBL_LDAP_GROUP_NAME' => 'Group Name:',
-    'LBL_LDAP_GROUP_NAME_DESC' => 'Example <em>cn=suitecrm</em>',
-    'LBL_LDAP_USER_DN_DESC' => 'Example: <em>ou=people,dc=example,dc=com</em>',
-    'LBL_LDAP_AUTO_CREATE_USERS_DESC' => 'If an authenticated user does not exist one will be created.',
-    'LBL_LDAP_ENC_KEY' => 'Encryption Key:',
-    'DEVELOPER_MODE' => 'Developer Mode',
-
-    'SHOW_DOWNLOADS_TAB' => 'Display Downloads Tab',
-    'LBL_LDAP_ENC_KEY_DESC' => 'For SOAP authentication when using LDAP.',
+	'SYSTEM_NAME'=>'System Name:',
```
Comment (horus68) [inline modules/Configurator/language/en_us.lang.php]: Isn't this new indentation and no space in '=>' a step back from the format already in use?
@Dillon-Brown Can you please reformat this file?

**Pull request merged**